locomotor/__init__.py

```diff
@@ -599,7 +605,7 @@ def process_Dict(self, node, code, indent, loops):
         """Generate code for a dictionary literal"""
 
         pairs = ["['__DICT'] = true"]
-        for key, value in itertools.izip(node.keys, node.values):
+        for key, value in itertools.zip_longest(node.keys, node.values):
```

Comment:
I don't understand this change. Why would `node.keys` and `node.values` ever be different lengths?